.github/workflows/trigger-styleguide-build.yaml

```diff
@@ -0,0 +1,20 @@
+name: Trigger styleguide build
+
+on:
+    push:
+        branches:
+            - master
+            - release/2.*
+
+jobs:
+    trigger-styleguide-build:
+        name: Trigger styleguide build
+        runs-on: ubuntu-latest
+        steps:
+            - name: Trigger styleguide build
+              uses: benc-uk/workflow-dispatch@v1
+              with:
+                  repo: sulu/sulu-javascript-docs
```

Comment:
would be nice to add a small `README` to the `sulu/sulu-javascript-docs` repo. otherwise looks good to me! 🙂

Reply:
fixed